compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -529,6 +529,13 @@ def PadDynamicAllocPass :
 def PolynomialApproximationPass :
     Pass<"iree-codegen-polynomial-approximation", ""> {
   let summary = "Convert math operations to their polynomial approximation";
+  let options = [
+    Option<"noApproxOps", "no-approx-ops", "std::string",
```

Comment:
Could we make this a proper list? `ListOption`

Reply:
Ahh thanks, didn't know about that!